source/extensions/omni.isaac.lab/omni/isaac/lab/terrains/terrain_generator_cfg.py

```diff
@@ -88,7 +88,7 @@ class SubTerrainBaseCfg:
     is 0.7.
     """
 
-    size: tuple[float, float] = MISSING
+    size: tuple[float, float] = (10.0, 10.0)
     """The width (along x) and length (along y) of the terrain (in m)."""
```

Comment:
```suggestion
    """The width (along x) and length (along y) of the terrain (in m). Defaults to (10.0, 10.0).
    
    In case the :class:`~omni.isaac.lab.terrains.TerrainImporterCfg` is used, this parameter gets overidden by
    :attr:`omni.isaac.lab.scene.TerrainImporterCfg.size` attribute.
    """
```